superset-frontend/src/views/CRUD/data/database/DatabaseModal.tsx

```diff
@@ -402,7 +419,7 @@ const DatabaseModal: FunctionComponent<DatabaseModalProps> = ({
             <div className="helper">
               {t('Refer to the ')}
               <a
-                href="https://docs.sqlalchemy.org/en/rel_1_2/core/engines.html#"
+                href={conf?.SQLALCHEMY_DOCS_URL ?? DEFAULT_SQLALCHEMY_DOCS_URL}
```

Comment:
per above, it seems that just conf.SQLALCHEMY_DOCS_URL should always have a value.

Reply:
do we want to also update the string text below?

Reply:
what string text are your referencing?

Reply:
the text for the link. "refer to the sqlalchemy docs...". It's in the ticket, too.